src/coreservices.cpp

```diff
@@ -156,6 +164,10 @@ void CoreServices::initializeLogging() {
 void CoreServices::initialize(QApplication* pApp) {
     ScopedTimer t("CoreServices::initialize");
 
+    VERIFY_OR_DEBUG_ASSERT(!m_isInitialized) {
```

Comment:
Move this early return before starting the timer. It acts like a precondition check that no other code should precede. Nitpicking.